pkg/skuba/actions/node/upgrade/apply.go

```diff
@@ -130,24 +130,20 @@ func Apply(client clientset.Interface, target *deployments.Target) error {
 		}
 	}
 
-	// Always upload crio files, regardless of the version (allows to
-	// enforce user behavior during patch updates).
-	// During the upgrade from 1.16 to 1.18 crio, the cri-o package will
-	// handle overriding the old crio sysconfig to an "empty" sysconfig,
-	// and the cri.configure action will be enough.
-	// We can remove the conditionals and only
-	// keep the cri.configure action when caasp 4.2.0 is not supported
-	// anymore (everyone has updated crio to 1.18)
-	if _, err := os.Stat(skuba.CriDefaultsConfFile()); err == nil {
-		err = target.Apply(nil, "cri.configure")
-		if err != nil {
-			return err
-		}
-	} else if _, err := os.Stat(skuba.CriDockerDefaultsConfFile()); err == nil {
-		err = target.Apply(nil, "cri.sysconfig")
-		if err != nil {
-			return err
-		}
+	// Always upload crio files, regardless of the version (allows to enforce
+	// user behavior during patch updates). During the upgrade from 1.16 to 1.18
+	// crio, the cri-o package will handle overriding the old crio sysconfig to
+	// an "empty" sysconfig, and the cri.configure action will be enough.
+	if _, err := os.Stat(skuba.CriDefaultsConfFile()); err != nil {
+		return err
```

Comment:
I am not sure if that error is very explicit. It will just say no such file or directory if the people have not upgraded their local config, right?

Should we be clearer?